SR2023/2023-01-04-horsham-feburary-tech-day.md

```diff
@@ -0,0 +1,34 @@
+---
+to: Student Robotics 2023 teams
+subject: Horsham February Tech Day
+---
+
+Hello,
+
+We're pleased to announce that we'll be hosting an [in-person Tech Day][event]
+on February 4th in Horsham. Full details can be found on the [event page][event].
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They’re also an opportunity to see how other
+teams are doing or get more direct help with your robots.
+
+The aim of this day is to help you out by having guaranteed presence from us
+blueshirts. We'll be around to answer questions about the competition, give kit
+support, and assist with your strategy ideas.
+
+We see a strong correlation between teams to attend Tech Days and performance in
+the competition.
+
+Spaces are limited to four teams, so be sure to [let us know][tech-day-signup] if
+you would like to attend. We will be confirming places shortly after you sign up.
+
+At the very latest please signup by Friday 24th February 2023, though we may
```

Comment:
Issue: This date is after the tech day

Reply:
[454d6fa](https://github.com/srobo/team-emails/pull/133/commits/454d6fab017121ba621ae615ffdafcc102d2ddf3)